KM review (part 2) -- Editorial pass on Section 6 (#120)
ietf-rats-wg/architecture: draft-ietf-rats-architecture.md

# Roles and Entities

An entity in the RATS architecture includes at least one of the roles defined
in this document.

> 
> These collapsed roles combine the duties of multiple roles.

Reviewer: ```suggestion
These collapsed roles combine the duties of multiple roles.

```

> in this document.
> 
> These collapsed roles combine the duties of multiple roles.

Reviewer: ```suggestion
in this document.
An entity can aggregate more than one role into itself. 
These collapsed roles combine the duties of multiple roles.

```

system bus Verifier, may choose to fully isolate its role as a wide-area
network Attester.

> In essence, an entity that combines more than one role may create and consume

Reviewer: ```suggestion
In essence, an entity that combines more than one role creates and consumes
```
